README.md

```diff
@@ -51,7 +51,12 @@ steps:
 
 ## Configuration
 
+### `api-secret-env` (Optional, string)
+
+The environment variable that the Wiz API Secret is stored in. Defaults to using `WIZ_API_SECRET`.
+
 ### `scan-type` (Required, string) : 'docker | iac'
```

Comment:
Is there a default scan type?

Reply:
Will also mention should update `plugin.yml` with the required configuration but that should be a separate PR
https://json-schema.org/draft/2020-12/json-schema-validation#name-required